From 9f1159a1a85e022717959b99ec3745f4c95543a7 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Sat, 14 Dec 2013 01:05:03 +0000
Subject: Fix a compiler warning

Don't return without a value from a non-void function.
---
(limited to 'gtk/gtkselection.c')

diff --git a/gtk/gtkselection.c b/gtk/gtkselection.c
index e20d464..caec0dd 100644
--- a/gtk/gtkselection.c
+++ b/gtk/gtkselection.c
@@ -2295,7 +2295,7 @@ _gtk_selection_request (GtkWidget *widget,
   gulong selection_max_size;
 
   if (event->requestor == NULL)
-    return;
+    return FALSE;
 
   if (initialize)
     gtk_selection_init ();
--
cgit v0.9.2
